docs/source/en/main_classes/text_generation.md

```diff
@@ -30,15 +30,17 @@ like token streaming.
 ## GenerationConfig
 
 [[autodoc]] generation.GenerationConfig
-	- from_pretrained
-	- from_model_config
-	- save_pretrained
-	- update
-	- validate
-	- get_generation_mode
+
+- from_pretrained
```

Comment:
I think the methods under `[[autodoc]]` should be indented so they get built correctly by the doc-builder

Reply:
I have checked locally with `doc-builder preview` and they were correctly rendered.